sktime/datatypes/_vectorize.py

```diff
@@ -312,7 +312,27 @@ def _get_item_indexer(self, i: int, X=None):
 
     def as_list(self):
         """Shorthand to retrieve self (iterator) as list."""
-        return list(self)
+        if (
+            isinstance(self.X_multiindex.index, pd.MultiIndex)
+            and self.iterate_as == "Series"
```

Comment:
would it make sense to also do this for `Panel`, and then just change the "-1" to "-2" below?

Reply:
Good idea.

Reply:
I see you factored this out into its own function. Good idea.